lua/autorun/sh_lastManStanding_init.lua

```diff
@@ -269,16 +269,16 @@ if CLIENT then
         end
     end)
 
-    net.Receive("lms_statistics_reveal",function()
-      if (net.ReadEntity() == LocalPlayer()) and (isfunction(StatisticsUpdatePData)) then
+    net.Receive("ttt_lms_reveal",function()
+      if (net.ReadBool()) and (isfunction(StatisticsUpdatePData)) then
         StatisticsUpdatePData("lms_Revealed", " revealed his/her role in total ", " times")
       end
     end)
 
-    net.Receive("lms_statistics_inno",function()
-      if (net.ReadEntity() == LocalPlayer()) and (isfunction(StatisticsUpdatePData)) then
+    net.Receive("ttt_lms_innocent",function()
+      if (net.ReadBool()) and (isfunction(StatisticsUpdatePData)) then
```

Comment:
I think net.ReadBool() can be removed. If you are not the player the ttt_lms_reveal net call is intended for, you will never receive this anyways.